src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -428,10 +431,16 @@ private int parseIota(FlowNode node) {
         System.out.println("Action format: ");
         System.out.println("\t- actionClassName actionDisplayName");
         System.out.println("------------------------------------------------------------------------------------------");
+        Function<FlowNode, String> flowNodeToId = new Function<FlowNode, String>(){
+            @Override
+            public String apply(@Nullable FlowNode input) {
+                return (input != null) ? input.getId() : null;
```

Comment:
Why are you checking for null? `FlowNode.getParents()` may not return null elements.